### PR TITLE
Makes the SD interface streamlined and adds timer to detonation

### PR DESCRIPTION
**Pull request by JackTheJackhammer**
## About The Pull Request

This pull request does 2 things:
1. it makes the SD interface more 'modern' and inline with most other interfaces in the game (see: shuttle interface)
2. adds a timer that displays time to detonation and a percentage bar

see:

## Why It's Good For The Game

People aught to have a easier way to see how much time is left on the SD. 
Muhrene not built for math, muhrene built for unga.

also it looks better

## Changelog

:cl:
qol: Reconfigured SD interface
add: Adds an indicator for how long the SD has left to activate.

/:cl:

### CODE REVIEW

**Comment by JackTheJackhammer** [comment]
1 check fail, gonna do in morning

File: code/game/objects/machinery/self_destruct.dm [hunk 1]
```diff
@@ -8,6 +8,10 @@
 	var/active_state = SELF_DESTRUCT_MACHINE_INACTIVE
 	///Whether only marines can activate this. left here in case of admins feeling nice or events
 	var/marine_only_activate = TRUE
+	///Tracks when the system was last armed
+	var/arm_time
+
+	var/list/temps = list()
```
Comment (TiviPlus) [inline code/game/objects/machinery/self_destruct.dm:14]: hi unused vars

**Review by TiviPlus — CHANGES_REQUESTED** [review]
rest lgtm

File: code/game/objects/machinery/self_destruct.dm (hunk 1 above)
Comment (JackTheJackhammer) [reply code/game/objects/machinery/self_destruct.dm:14]: damned

**Pull request merged**